pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	machineAnnotationKey          = "machine.openshift.io/machine"
-	machineRebootAnnotationKey    = "healthchecking.openshift.io/machine-remediation-reboot"
```

Comment:
can you also change `healthchecking.openshift.io` -> `machine.openshift.io`

Reply:
wouldn't you want to chance the variable name as well? and the func `remediationStrategyReboot` name and its events and logging?

Reply:
good point, I'll do that too

Reply:
also `healthchecking.openshift.io/strategy` -> `machine.openshift.io/remediation-strategy` ?